src/fetch_agent/fetch_agent/agent.py

```diff
@@ -0,0 +1,58 @@
+import rclpy
+from rclpy.node import Node
+
+from std_msgs.msg import String
+
+from uagents import Agent, Context
+from uagents.setup import fund_agent_if_low
+
+from aca_protocols.station_query_protocol import (
+    StationQueryRequest,
+    StationQueryResponse,
+)
+
+from aca_protocols.acs_registry_id import acs_id
+
+agent = Agent(
+    name="car",
+    seed="Car1",
+    port=8002,
+    endpoint=["http://127.0.0.1:8002/submit"],
+)
+
+fund_agent_if_low(agent.wallet.address())
+
+
+@agent.on_event("startup")
+async def introduce_agent(ctx: Context):
+    ctx.logger.info(f"Agent: {agent.name} ({agent.address})")
+    await ctx.send(
+        acs_id,
+        StationQueryRequest(lat=1.0, long=1.0, radius=5.0),
```

Comment:
Is this for dev or why should a car query stations on startup?

Reply:
yea, i just copied the stub from acs-registry to test if it works